sample_app/sampleapp_common.c

```diff
@@ -427,6 +538,73 @@ static int app_read_ind (
    return 0;
 }
 
+static int app_set_data_and_ioxs(
+      pnet_t * net,
+      app_data_t * p_appdata,
+      app_api_t *p_api,
+      uint16_t slot,
+      uint16_t subslot_index)
```

Comment:
Maybe `slot_number` and `subslot_number`?

Reply:
The subslot index is easy to mix up with the subslot number. I changed the function prototype to use app_subslot_t instead to avoid this problem.
Done